<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>emailyanshi</class>
 <widget class="QDialog" name="emailyanshi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>751</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextEdit" name="textEdit_Text">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>295</y>
     <width>431</width>
     <height>101</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>邮件内容......</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backbutton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="logid_lineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>431</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>账号......</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>461</width>
     <height>731</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2"/>
  </widget>
  <widget class="QLineEdit" name="Rcvemail_lineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>125</y>
     <width>431</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>接收邮箱......</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="logpasswd_lineEidt">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>175</y>
     <width>431</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>授权码......</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="server_textEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>455</y>
     <width>431</width>
     <height>271</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>服务器反馈......</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="subject_lineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>225</y>
     <width>431</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>发送邮件标题......</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sendButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>410</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="SendEmail_lineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>431</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>发送邮箱......</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>-5</x>
     <y>1</y>
     <width>511</width>
     <height>761</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>511</width>
     <height>761</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/background.png);</string>
   </property>
  </widget>
  <zorder>graphicsView</zorder>
  <zorder>textEdit_Text</zorder>
  <zorder>logid_lineEdit</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>Rcvemail_lineEdit</zorder>
  <zorder>logpasswd_lineEidt</zorder>
  <zorder>server_textEdit</zorder>
  <zorder>subject_lineEdit</zorder>
  <zorder>sendButton</zorder>
  <zorder>SendEmail_lineEdit</zorder>
  <zorder>backbutton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
